bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -100,7 +100,7 @@ def load_mass_data(input_file_handle, key, tag):
     return data
 
 
-def load_sky_error_data(input_file_handle, key, tag, trig_data):
+def load_sky_error_data(input_file_handle, key, tag):
```

Comment:
Unless I am missing something, we want to keep injections and trigger separated, hence the need for `input_file_handle` and `trig_data`.

Reply:
The `trig_data` in this code contains the triggers from the no-injections side of the workflow. The found-missed injections file (`input_file_handle` here) contains the triggers associated with found injections along with the parameters of those injections.

The error is that it was trying to compare different sets of triggers, which are not necessarily the same size so index errors can happen.

Reply:
Can you post an example of how you get the error you are referring to?  I'm trying to understand why this error was never encountered before.

This function is meant to calculate the sky errors by considering the injected RA and dec values and the recovered RA and dec values.  So `inj` needs to contain the injected values, while `trig` the recovered ones and we need to ensure we are comparing them correctly (i.e., the recovered values must correspond to the correct injection).

I think we may have to open a separate issue for this.

Reply:
Ok, I'll open a new PR for this and put together a case where the error can be seen. It's now been removed from this PR.